docs/contributors/contributors-guide.md

```diff
@@ -160,16 +166,17 @@ There are many things you can do with this feature, but here are some ideas to g
 
 - **Track personal contributions**: If you create a list of your personal contributions, you can track them and see how you're doing over time, as well as get a breakdown of the type of contributions you're making, including commits, created PRs, reviewed PRs, and created issues.
 - **Learn about what other contributors are doing in open source**: You can learn where other contributors are contributing, what they're working on, and how active they are in the open source community.
-- **See who's using OpenSauced Highlights feature**: You can see who's using the Highlights feature and show them support by giving an emoji reaction.
+- **See who's using the OpenSauced Highlights feature**: You can see who's using the Highlights feature and show them support by giving an emoji reaction.
```

Comment:
[improvement] Here, it would be beneficial to point out a aspect of open source & how the OpenSauced feature can facilitate it. What do you think @adiati98?
```suggestion
- **Support other contributors**: Networking with others is key to thriving in open source. You can do this by posting a emoji  on someone’s highlight in the Highlights section.
```

Reply:
@CBID2 that's out of the scope of this PR.